.github/workflows/gh-ci.yaml

```diff
@@ -0,0 +1,103 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+concurrency:
+  # Specific group naming so CI is only cancelled
+  # within same PR or on merge to main
+  group: "${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  main-tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.8", "3.9", "3.10"]
+        installer: ["pip", "mamba"]
+        include-tests: [true, false]
+        mdanalysis-version:
+          - install-version: develop
+            expected-version: "2.3.0-dev0"
+          - install-version: latest
+            expected-version: "2.2.0"
+          - install-version: "2.1.0"
+            expected-version: "2.1.0"
+
+        include:
+          - os: windows-latest
+            python-version: "3.9"
```

Comment:
To get spread

Reply:
It's up to you, and you don't need to fix this now even if you wanted to. 

I think we can get away with saying that spread is covered by the matrix above. For the sake of the person coming along to fix this, it might make more sense to just assume that the only variable (the os version) is being toggled here.

Reply:
I suppose so -- but another way to think about it is that just the OS version is toggled from all the Ubuntu tests. I'll change it to 2.1.0 so it's covered.

Reply:
Can you at least add a comment somewhere saying "lowest python and MDA versions + windows test"? That way future folks will come along and know the intent of why things were chosen as such?